com.vmware.horizon/com.vmware.horizon.yaml

```diff
@@ -0,0 +1,86 @@
+---
+app-id: com.vmware.horizon
+default-branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: vmware-view
+finish-args:
+  - "--share=ipc"
+  - "--device=dri"
+  - "--socket=x11"
```

Comment:
this doesn't work under wayland? Since it seems to use gtk3 for the UI, I'd expect it to.

Reply:
This bit is in their wrapper script they have though more testing is nessecary. 

```sh
# Add path to LD_LIBRARY_PATH
vm_append_to_library_path() {
   local ldBasePath
   if [ ! -z "$LD_LIBRARY_PATH" ]; then
      ldBasePath="$LD_LIBRARY_PATH"
   fi
   export LD_LIBRARY_PATH="$1:""$ldBasePath"
   # We use x11 lib in client, so cannot use wayland now
   export GDK_BACKEND=x11
}
```

Reply:
ah ok.